### PR TITLE
Refactor #3527 : remove jquery-ui and outdated code

### LINKED ISSUE #3527: Remove jQuery-UI script
### Describe the bug

I found in some files still contain the jquery-ui, It should be removed.
- interface/usergroup/facilities_add.php
- interface/forms/eye_mag/help.php

I will remove it, Please assign this issue to me.

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3527 

#### Short description of what this resolves:
Remove jquery-ui and outdated code.

### CODE REVIEW

File: interface/forms/eye_mag/help.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="description" content="Eye Exam Help" />
     <meta name="author" content="openEMR: ophthalmology help" />
-    <?php Header::setupHeader(['jquery-ui', 'jquery-ui-excite-bike']); ?>
+    <?php Header::setupHeader(); ?>
```
Comment (stu01509) [inline interface/forms/eye_mag/help.php:38]: Hello @bradymiller and @tywrenn 
I would like to use Bootstrap collapse components. But it seems like Bootstrap js didn't load completely.

image

Comment (tywrenn) [reply interface/forms/eye_mag/help.php:38]: JS shouldn't be required to use Bootstrap's accordion collapse components. This JS might need to be looked at again or replaced entirely.

Comment (stu01509) [reply interface/forms/eye_mag/help.php:38]: @tywrenn  Yup, I found in my codebase has a little bit code conflict that caused this issue. I will clone the new project after I finished #3497

**Comment by stu01509** [comment]
I created a new PR #3569, So I close this.

**Pull request closed without merging**